src/components/TodoFooter/TodoFooter.tsx

```diff
@@ -0,0 +1,68 @@
+import cn from 'classnames';
+import { Status } from '../../types/Status';
+import { Todo } from '../../types/Todo';
+
+interface Props {
+  allTodos: Todo[]
+  visibleTodos: Todo[]
+  status: Status
+  onStatus: (status: Status) => void
+  onClear: () => void
+}
+
+export const TodoFooter: React.FC<Props> = ({
+  allTodos, visibleTodos, status, onStatus, onClear,
+}) => {
+  return (
+    <footer className="todoapp__footer" data-cy="Footer">
+      <span className="todo-count" data-cy="TodosCounter">
+        {`${allTodos.filter(todo => !todo.completed).length} items left`}
```

Comment:
Such a record is very unreadable, put the calculation of the number in a separate variable